modin/engines/base/io/text/csv_dispatcher.py

```diff
@@ -72,7 +80,7 @@ def _read(cls, filepath_or_buffer, **kwargs):
                 filepath_or_buffer,
                 **dict(kwargs, usecols=None, nrows=0, skipfooter=0, index_col=None),
             ).columns
-        elif index_col is None and not usecols:
+        elif index_col is None and not kwargs.get("usecols", None):
```

Comment:
You already define `usecols` variable, why not used here?

Reply:
Right, modified `usecols` can be used here. Fixed.